Add page: Prebid Analytics by Roxot.

kir-roxot opened the pull request:
Hey guys.

We added a page with a description of Prebid Analytics by Roxot and workflow examples. Could you merge it to the master as soon as the 0.22.0 prebid.js is released? This prebid version has our analytics adapter.

kir-roxot commented:
@mkendall07  
Hi!
Could you review the code? Let us know if there are any issues. Also, could you implement these changes to the website update devoted to the release of prebid.js 0.22? Thanks in advance.

bretg commented:
This is the first doc that a vendor's added to the Prebid site. My initial take is that we may want to carve out a separate "marketplace" space for this type of thing. 

We plan on adding a general "analytics adapter" page that outlines the options available to publishers, but additional information like this has a place -- just need to hammer out the structure.

kir-roxot commented:
@bretg  Let us know if we can help speed up the process. Thanks.

GLStephen commented:
@bretg Stephen from http://www.PubWise.io here. We'd like to participate however we can contribute to any process around this and we've just submitted our adapters and Prebid testing components to the Prebid.js project. Please let us know how we can contribute.

kir-roxot commented:
@bretg 
Hey Bret! We have updated the PR to reflect the changes you mentioned in your comments. We toned down the first paragraph, added installation instructions with an example of code to put into the page. Please, review.

bretg commented:
Thanks @kir-roxot  - this is definitely an improvement. Am reviewing with the Prebid Org team. Your continued patience appreciated as there are several key stakeholders traveling.

@GLStephen - will contact you via your company's support email.

rmloveland commented:
Hey @kir-roxot, as you probably already know we're moving away from vendor-specific analytics integration instructions (e.g., https://github.com/prebid/prebid.github.io/issues/297), and we've added an [Analytics Overview page](http://prebid.org/overview/analytics.html) that links to Roxot and other vendors' solutions.

Therefore I'm going to close this PR.  Thanks for the submission though, and I apologize for the long, drawn out process to get to a response on this!
